src/libraries/System.Text.Json/Directory.Build.props

```diff
@@ -3,5 +3,6 @@
   <PropertyGroup>
     <StrongNameKeyId>Open</StrongNameKeyId>
     <PackageVersion>5.0.1</PackageVersion>
+    <AssemblyVersion Condition="'$(TargetFramework)' == 'net461'">5.0.0.1</AssemblyVersion>
```

Comment:
Is there a way to test this (even one-off/locally) to make sure other `assembly version` attributes aren't missing for some supported TFM?

Reply:
@ahsonkhan can you clarify what you meant here?  `NetFramework` is the only place where we must change the version (due to GAC).  In 5.0 we are avoiding version changes for any assembly that is also part of the shared framework (and compatible TFMs like netstandard) so that we can have servicable ref-packs.

Reply:
> `NetFramework` is the only place where we must change the version (due to GAC)

I understand this is needed for .netfx only. The question I was trying to ask is, how do we verify that this change is correct and that we are not missing other assemblies that would need a similar change.

> json change was missing the assembly version update for netfx

For example, how did we detect this originally?

Reply:
@Anipik reviews the changes that are made in servicing to ensure that they have both packaging and assembly changes since this is something unique to servicing.  This is part of the "decision" to service the assembly, you can think of it as the change that represents the decision to opt something into shipping.  We're aware that representing this decision requires more changes than we'd like and are thinking of ways to streamline that.  Ultimately there will always be some change like this that says: increment a version and ship. We'd like to make it much simpler so that it's literally just that and all the conditions and special cases are handled by the build instead of the developer.